src/api/v2/agents/provision/handlers/provision-sms.ts

```
@@ -0,0 +1,101 @@
+    return;
+  }
+
+  const { instanceId } = parsed.data;
+  req.log.info({ instanceId }, "SMS provision request received");
+
+  try {
+    const poolBaseUrl = AGENT_POOL_URL.replace(/\/+$/, "");
+
+    const poolRes = await fetch(
+      `${poolBaseUrl}/api/proxy/sms/provision`,
+      {
```

ESLint

[error] Replace `⏎······`${poolBaseUrl}/api/proxy/sms/provision`,⏎·····` with ``${poolBaseUrl}/api/proxy/sms/provision`,`

(prettier/prettier)